opencog/guile/SchemeSmob.cc

```diff
@@ -232,6 +232,9 @@ void SchemeSmob::register_procs(void*)
 	register_proc("cog-ptv?",              1, 0, 0, C(ss_ptv_p));
 	register_proc("cog-ftv?",              1, 0, 0, C(ss_ftv_p));
 	register_proc("cog-tv->alist",         1, 0, 0, C(ss_tv_get_value));
+	register_proc("cog-tv-mean",           1, 0, 0, C(ss_tv_get_mean));
+	register_proc("cog-tv-confidence",     1, 0, 0, C(ss_tv_get_confidence));
+	register_proc("cog-tv-count",          1, 0, 0, C(ss_tv_get_count));
```

Comment:
I like all of the rest of this patch, except for this. It would be much better to implement this in scheme code, 

The utilities.scm file already implements tv-mean and tv-conf and tv-count. You can rename these to cog-tv-mean etc if you wish.  They take a TV by default, but maybe you can implemnt a version that takes atoms directly.